php/Blockonomics.php

```diff
@@ -400,6 +400,10 @@ public function is_lite_mode_active(){
         return get_option('blockonomics_lite', false);
     }
 
+    public function is_partial_payments_active(){
+        return get_option('blockonomics_partial_payments', false);
```

Comment:
Since we are setting the default to `true`, we may want to use the default as `true` here as well. This will only affect users who have not saved the Blockonomics settings, but have activated the plugin and using it.